Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/MacroConstants.kt

```diff
@@ -133,53 +130,27 @@ private fun processCodeSnippet(
 ): ConstantDef? {
 
     var state = VisitorState.EXPECT_VARIABLE
-    var evalResultKind: CXEvalResultKind? = null
-    var type: Type? = null
-    var longValue: Long? = null
-    var doubleValue: Double? = null
+    var evalResultOrNull: CXEvalResult? = null
```

Comment:
WDYT about lateinit and isInitialized here?

Reply:
I've tried to use it and discovered that local lateinit doesn't support `isInitialized`.